src/utils/sources-tree.js

```diff
@@ -163,7 +163,7 @@ function isDirectory(url: Object) {
  * @memberof utils/sources-tree
  * @static
  */
-function addToTree(tree: any, source: TmpSource) {
+function addToTree(tree: any, source: TmpSource, debuggeeUrl: any) {
```

Comment:
Is the URL really an any? I don't mind using it but if there's a type we could use it's preferable.